cmake_format/test/cmake-format.json

```diff
@@ -17,4 +20,4 @@
       }
     }
   }
-}
+}
```

Comment:
@dkandyba please add empty line.

Reply:
@BSolonenko I think this changes is redundant. What is the reason adding empty line?

Reply:
@BSolonenko  Fixed in: dc880b1cb71405b990ba4615189e39ccbd14f6a6